test/functional/test-activity.js

```diff
@@ -140,6 +140,77 @@ describe('Activity getTotalEngagedTime', () => {
     return expect(activity.viewer_).to.equal(viewer);
   });
 
+  it('should have 0 incremental engaged time if there is no activity', () => {
```

Comment:
These tests are a copy of tests against `#getTotalEngagedTime`, which is not the test we want. 

My suggestion: 3 things to test:
1. Test that when `#getIncrementalEngagedTime()` gets called for first time, it return the correct value.
2. Test that when `#getIncrementalEngagedTime()` gets called multiple times, it return the incremental time, not the total time.
3. Test that when `#getIncrementalEngagedTime()` gets called with different `name`, it return the corresponding incrementa.

Thanks.